doc/sphinx/userguide/python-tutorial.md

```diff
@@ -199,9 +199,8 @@ Cantera search path:
 ct.add_directory('~/cantera/my_data_files')
 ```
 
-% TODO: Add link to input file guide
 Cantera input files are plain text files, and can be created with any text editor. See
-the document `Working With Input Files` for more information.
+the page [](input-tutorial) for more information.
```

Comment:
Is this a typo? (empty `[]`). Similar below.

Reply:
No, this is the syntax for automatically generating the link text based on the title of the linked page.